clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClientBuilder.java

```diff
@@ -175,6 +175,9 @@ ZeebeClientBuilder jobWorkerExecutor(
    */
   ZeebeClientBuilder defaultJobWorkerStreamEnabled(boolean streamEnabled);
 
+  /** If enabled, the client will make use of the default retry policy defined. False by default. */
+  ZeebeClientBuilder useDefaultRetryPolicy(final boolean useDefaultRetryPolicy);
```

Comment:
❓ Should we elaborate about this default retry policy? Perhaps refer to the current default policy definition.

Reply:
Once I've updated the docs, we can add the link here I think. For now I do a poor man's link of saying the file is in which module =/